app/workers/deletes.php

```diff
@@ -95,6 +98,23 @@ protected function deleteUser(Document $document, $projectId)
         ], $this->getProjectDB($projectId));
     }
 
+    protected function deleteExecutionLogs(Document $document) 
+    {
+        
+        $projectIds = $document->getAttribute('projectIds', []);
+        foreach ($projectIds as $projectId) {
+            if (!($projectDB = $this->getProjectDB($projectId))) {
+                throw new Exception('Failed to get projectDB for project '.$projectId, 500);
+            }
+
+            // Delete Executions
+            $this->deleteByGroup([
+                '$collection='.$document->getCollection(),
```

Comment:
What document is passed here? Maybe we should use Database::SYSTEM_COLLECTION_EXECUTIONS to avoid any mistakes that could lead to data loss?

Reply:
![Screenshot 2020-12-17 at 9 08 15 PM](https://user-images.githubusercontent.com/20852629/102508941-fb062000-40ab-11eb-873f-4f811379d4ec.png)

This is the document passed

Reply:
We discussed changing the way we trigger that worker anyway... So this is probably not relevant anymore.

Reply:
sure 👍